File: bmi.cpp
#include <iostream>
#include <cmath>

using namespace std;

class Bodymass{
    float w, h;
public:
    void set_values(float,float);
    float BMI() {return (w/pow(h/100,2.0));}
};

void Bodymass::set_values(float x,float y){
    w=x;
    h=y;
    }

int main()
{
    Bodymass person;
    float ww,hh;
    cout << "Set weight(kg): ";
    cin >> ww;
    cout << endl;
    cout <<"Set height(cm): ";
    cin >> hh ;
    person.set_values(ww,hh);
    std::cout.precision(4);
    cout <<"Your BMI is: "<<person.BMI() <<endl;
    return 0;
}

File: bmi.py
import math

class Bodymass:
    def __init__(self,weight,height):
        self.weight=weight
        self.height=height
    def BMI(self):
        return self.weight/pow(self.height,2.0)


try:
    w=float(input("Weight is: "))
    h=float(input("Height is: "))
except:
    print("Input must be a number")
person=Bodymass(w,h)
#print(person.weight)
#print(person.height)
print("Your BM index is: ",round(person.BMI(),2))
